hack/generate-metallb-manifests.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 . $(dirname "$0")/common.sh
 
-METALLB_COMMIT_ID="d7afd9c94e246bc838b49a5fdee5384528513d63"
+METALLB_COMMIT_ID="24b8f9c349866e200ddeda1052b2038f21098b13"
```

Comment:
when we update this commit we will have to update the commit in the CI lane as well?
maybe it can be stored in a file?

Reply:
sorry missed that it is done just after.